### DISCUSSION TITLE
Undo and redo's for selections

### DISCUSSION THREAD

**Discussion by Landy005** [Mojang/minecraft-editor#52] [Feedback]
Selections aren't registered in undo's and redo's. 

If I select an area and change the selection I'd naturally think hitting undo would get me back my previous selection box area. But instead it undo's the last world operation.

**Comment by CaptainCooky** [comment]
I'd think this would have to be a separate feature, with it's own keybind/shortcut. 
It is needed though👍

**Comment by minecraftchloe** [comment]
I'm so happy to see this feedback - it's been on my personal wishlist from the beginning! 😅
